graph/graph.py

```diff
@@ -23,13 +24,65 @@ class KytosGraph:
 
     def __init__(self):
         self.graph = nx.Graph()
+        self._accepted_metadata = {
+            'ownership',
+            'bandwidth',
+            'reliability',
+            'priority',
+            'utilization',
+            'delay',
+        }
+        ownership_processor = ProcessEdgeAttribute(
+            'ownership',
+            TypeDifferentiatedProcessor({
+                str: lambda x: frozenset(x.split()),
+                dict: lambda x: frozenset(x.keys()),
+                type(None): None
+            })
+        )
         self._filter_functions = {
-            "ownership": lazy_filter(str, filter_in("ownership")),
-            "bandwidth": lazy_filter((int, float), filter_ge("bandwidth")),
-            "reliability": lazy_filter((int, float), filter_ge("reliability")),
-            "priority": lazy_filter((int, float), filter_le("priority")),
-            "utilization": lazy_filter((int, float), filter_le("utilization")),
-            "delay": lazy_filter((int, float), filter_le("delay")),
+            "ownership": EdgeFilter(
+                operator.and_,
```

Comment:
@Ktmi, "ownership" accepting multiple and including any intersection still needs to be confirmed if indeed it should behave like a `any` or `all`, currently it's behaving like `any`. So if users want to include `{"mandatory_metrics": {"ownership": "red,blue"}}`, what's the expected behavior? Both ways could make sense, but we need to sort out this requirement with network engineers, and then document in the openapi.yml doc accordingly.

Reply:
Moved ownership to just accept a single, rather than deal with the multiple cases, as discussed with @italovalcy.

Reply:
Right. Let's close this thread then.